modules/ldsg.js

```diff
@@ -0,0 +1,147 @@
+    const response = `📈 **Members:**\n${ldsg.memberCount} Members\n${online.size} Online`;
+    await interaction.reply({ content: response });
+  } catch (error) { u.errorHandler(error, interaction); }
+}
+
+const sendOptions = [
+  u.MessageActionRow().setComponents([
+    new u.Button().setCustomId("pa").setEmoji("🗣️").setLabel("Public Affairs").setStyle(ButtonStyle.Success),
+    new u.Button().setCustomId("log").setEmoji("🚚").setLabel("Logistics").setStyle(ButtonStyle.Success),
+    new u.Button().setCustomId("ops").setEmoji("🗓️").setLabel("Operations").setStyle(ButtonStyle.Success),
+    new u.Button().setCustomId("mgmt").setEmoji("⚙️").setLabel("Management").setStyle(ButtonStyle.Primary),
+    new u.Button().setCustomId("icarus").setEmoji("🤖").setLabel("Icarus").setStyle(ButtonStyle.Primary),
+  ]),
+  u.MessageActionRow().setComponents([
+    new u.Button().setCustomId("ignore").setEmoji("⚠️").setLabel("Ignore").setStyle(ButtonStyle.Danger),
+  ])
+];
+
+const replyOption = [
+  u.MessageActionRow().setComponents([
+    new u.Button().setCustomId("reply").setEmoji("🗨️").setLabel("Reply to user").setStyle(ButtonStyle.Primary)
+  ])
+];
+
+/** @param {Discord.ChatInputCommandInteraction} int */
+async function slashLdsgSuggest(int) {
+  const suggestion = int.options.getString("suggestion", true);
+  await int.deferReply({ ephemeral: true });
+  const embed = u.embed({ author: int.user })
+    .addFields({ name: "User ID:", value: int.user.id }, { name: "Suggestion:", value: suggestion });
+  await int.client.getTextChannel(u.sf.channels.suggestionBox)?.send({ embeds: [embed], components: sendOptions });
+  int.editReply("Sent!");
+  return int.user.send({ content: "Sent:", embeds: [embed] });
+}
+
+/** @param {Discord.ButtonInteraction<"cached">} int */
+async function processCardAction(int) {
+  try {
+    const suggestion = int.message;
+    const embed = u.embed(suggestion.embeds[0]);
+    const modal = new u.Modal().addComponents(
+      u.ModalActionRow().addComponents([
+        new u.TextInput()
+          .setCustomId("text")
+          .setLabel("Reply")
+          .setStyle(Discord.TextInputStyle.Short)
+          .setRequired(true)
+          .setPlaceholder("Your reply for the user")
+      ])
+    ).setCustomId("suggestionReply").setTitle("Suggestion Reply");
+    let submitted;
+    let em;
+    let reply;
+    let member;
+    /** @type {String} */
+    let channel;
+    switch (int.customId) {
+      case "pa":
+        channel = u.sf.channels.publicaffairsForum;
+        break;
+      case "log":
+        channel = u.sf.channels.logisticsForum;
+        break;
+      case "ops":
+        channel = u.sf.channels.operationsForum;
+        break;
+      case "icarus":
+        channel = u.sf.channels.bottestingForums;
+        break;
+      case "mgmt":
+        channel = u.sf.channels.managementForum;
+        break;
```

Comment:
Consider something like this:
```js
const channel = {
  pa: u.sf.channels.publicaffairsForum,
  log: u.sf.channels.logisticsForum
  ...
}[int.customId]
```
and using something like `if (channel) { ... return } else if (customId == 'reply') { ... return } else { ... return }`